docs/requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.12
```

Comment:
Do these still work with 3.8? This might be missing from our docs, but we generally need to update the requirements with the readthedocs version (or maybe that is 3.12 now?), to make sure we aren't picking up something not available. Having three dependencies vanish makes me wonder if they are only needed in older Python.

Reply:
I tested with 3.8 and it worked, but to reduce the scope of this PR I'm reverting this back to 3.8.